### PR TITLE
Adds a data generation script

### PR DESCRIPTION
**Pull request by brockross**
Script generates x entries @ y entries per file, depending on arguments passed when the CLI command is run (i.e., `npm run gen-data 10000 1000` would generate 10k entries, with 1k entries in each new file). Entries are in JSON format

### CODE REVIEW

File: dataGenerator.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    fs.writeFile(`./data/testData${fileNameSerial}.json`, JSON.stringify(dataList), (err) => {
+      if (err) {
+        console.log(err);
+      } else {
+        console.log(`saved file ${notifySerial}`);
+      }
+    });
+  }
+}
+
```
Comment (sydneycasey) [inline dataGenerator.js:71]: For this file, I would just say clean up the comments a bit if they aren't helpful / necessary (it appears that most of them are but some seem like they may be unnecessary). Nice use of console.logs! The variable `notifySerial` kind of threw me off, but after reading through it again it looks like you're just using it to keep track of which file is being written at what time. I might change the name to something more descriptive.

**Review by sydneycasey — COMMENTED** [review]
I left a comment on dataGenerator.js, other than a few small details it looks good to me! 👍

File: dataGenerator.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+const faker = require('faker');
+const fs = require('fs');
+// const bson = require('bson');
+// const loremHipsum = require('lorem-hipsum');
+
+let dataList = [];
+
+//number of entries to generate -- defaults to 100k if not provided as arg
+const entryQty = process.argv[2] || 100000;
```
Comment (kazshige) [inline dataGenerator.js:9]: Using the command line argument looks really nice. Just wondered if you created 100 files, next you would need to seed those 100 files to your databases. Do you think the scripts' sounds performance efficient?

**Review by kazshige — COMMENTED** [review]
Overall, everything looks great. Just commented on one thing.

**Pull request merged**